site-pages/sign-up/sign.css

```diff
@@ -0,0 +1,35 @@
+.signup-main{
+    display: flex;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    align-items: center;
+    justify-content: center;
+}
+.signup-form-container{
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    align-self: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 20px;
```

Comment:
Can use rems here too

Reply:
I totally forgot about changing px to rem. Will do that.